aws/elastic-beanstalk/blueprint.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: xl/v2
+kind: Blueprint
+
+metadata:
+  name: AWS-Elastic-Beanstalk-Sample
+  description: |
+    The blueprint deploys a sample AWS Elastic Beanstalk PHP5 application.
+  author: XebiaLabs
+  version: 2.0
+  instructions: Please read the generated file "xebialabs/USAGE.md" for further usage instructions.
+
+spec:
+  parameters:
+
+  - name: AppName
+    type: Input
+    prompt: What is the name of the application?
+
+  - name: S3BucketName
+    type: Input
+    prompt: What is the name of the S3 bucket where CloudFormation artifacts should be saved?
+    validate: !expr "regex('^(([a-z0-9]|[a-z0-9][a-z0-9\\\\-]*[a-z0-9])\\\\.)*([a-z0-9]|[a-z0-9][a-z0-9\\\\-]*[a-z0-9]){3,63}$', S3BucketName)"
+    default: beanstalk-sample
```

Comment:
May be we can use `!expr " AppName + '-beanstalk-sample' "` to avoid conflicts and make default more useful?

Reply:
Done